iothub/service/src/IotHubServiceClient.cs

```diff
@@ -233,6 +234,9 @@ private void InitializeSubclients(IotHubServiceClientOptions options)
             MessageFeedback = new MessageFeedbackProcessorClient(_hostName, _credentialProvider, options, _retryHandler);
             FileUploadNotifications = new FileUploadNotificationProcessorClient(_hostName, _credentialProvider, options, _retryHandler);
 
+            // Specify the JsonSerializerSettings for subclients
+            JsonConvert.DefaultSettings = JsonSerializerSettingsInitializer.GetJsonSerializerSettingsDelegate();
```

Comment:
What about the other clients? Would you need to do this for provisioning clients as well? The device client might need a slightly different approach since the serializer there is customizable.

Reply:
Good point! I will look into other clients within a new PR since I missed this for dps clients in the previous PR (commit to "main").